README.md

```diff
@@ -153,7 +153,10 @@ The following tables lists the configurable parameters of the chart and their de
 | datadog.datadog.clusterName                                                | Name of EKS cluster                                                                   |                                                                                     |
 | datadog.enabled                                                            | Enables datadog                                                                       | FALSE                                                                               |
 | graderSetupService.enabled                                                 | Enables Grader Setup Service                                                          | FALSE                                                                               |
-| graderSetupService.graderImage                                             | Grader Image Name                                                                     | illumidesk/illumidesk-grader:latest                                                 |
+| graderSetupService.graderSpawnerImage                                      | Grader Image Name                                                                     | illumidesk/illumidesk-grader:latest                                                 |
+| graderSetupService.graderSpawnerCPU                                        | CPU Allocated for each grader                                                         | 200m                                                                                |
+| graderSetupService.graderSpawnerMem                                        | Memory Allocated for each grader                                                      | 400Mi                                                                                |
+| graderSetupService.graderSpawnerStorage                                    | Storage Allocated for each grader                                                     | 500Mi                                                                                 |
```

Comment:
@Abhi94N I'm about to create a PR in the `illumidesk/illumidesk` repo to add support for requests and limits for CPU/Memory so we should go ahead and include the option to update the variables:

- graderSpawnerMemGuarantee
- graderSpawnerMemLimit
- graderSpawnerCpuGuarantee
- graderSpawnerCpuLimit